docs/api/nodes/element.md

```diff
@@ -30,6 +31,25 @@ A "block" element can only be siblings with other "block" elements. An "inline"
 
 In a not "void" element, Slate handles the rendering of its `children` (e.g. in a paragraph where the `Text` and `Inline` children are rendered by Slate). In a "void" element, the `children` are rendered by the `Element`'s render code.
 
+#### Rendering Void Elements
+
+Void Elements must
```

Comment:
Either remove must here or the start of each list item :)

Reply:
Good catch!